eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -99,5 +99,17 @@ func (c *ClusterLoader) buildConfigMapData() (map[string]string, error) {
 }
 
 func (c *ClusterLoader) buildArgs() string {
-	return fmt.Sprintf("[%s]", strings.Join(c.Config.Spec.ClusterLoader.TestParams, ","))
+	return fmt.Sprintf("[%s]", strings.Join(append(c.Config.Spec.ClusterLoader.TestParams, "--report-dir=/usr/share/cluster-loader"), ","))
```

Comment:
This is going to be marginally difficult to extend for future engineers. I might do something like:
```
arguments := c.Config.Spec.ClusterLoader.TestParams
for _, argument := []string{
  "--report-dir=/usr/share/cluster-loader",
} {
  arguments = append(arguments, argument)
}
return fmt.Sprintf("[%s]", strings.Join(arguments, ","))
```

Reply:
Ah! Good idea.

Reply:
I'm going to leave as is. The appending argument isn't something that's going to need changing.